bin/node-template/runtime/src/lib.rs

```diff
@@ -207,6 +207,7 @@ impl pallet_aura::Config for Runtime {
 	type DisabledValidators = ();
 	type MaxAuthorities = ConstU32<32>;
 	type AllowMultipleBlocksPerSlot = ConstBool<false>;
+	type SlotDuration = pallet_aura::MinimumPeriodTimesTwo<Runtime>;
```

Comment:
I added this just to make the doc-CI happy (as it builds with all workspace features)